lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -499,12 +499,13 @@ def _init_gridlines(self, grid_helper=None):
         # It is done inside the cla.
         self.gridlines = self.new_gridlines(grid_helper)
 
-    def cla(self):
-        # gridlines need to b created before cla() since cla calls grid()
+    def clear(self):
+        # docstring inherited
+        # gridlines need to b created before clear() since clear calls grid()
```

Comment:
```suggestion
        # gridlines need to be created before clear() since clear calls grid()
```
Might as well fix the typo while we're at it.

Reply:
Fixed